docs/website/generator-errors.md

```diff
@@ -198,6 +198,10 @@ This usually indicates a bug in Xamarin.iOS/Xamarin.Mac; please [file a bug repo
 
 ### <a name='BI1066'/>BI1066: Unsupported return type '{type}' in {method}.
 
+### <a name='BI1067'/>BI1067: BindAs attribute of the selector \"{selector}\" is missing '[]' in the definition.
+
+### <a name='BI1068'/>BI1068: BindAs attribute of the parameter \"{parameter}\" is missing '[]' in the definition.
```

Comment:
`The BindAs type for the parameter "{parameterName}" in the method "{type}.{method}" must be an array when the parameter's type is an array.`

Note that I also include type and the method name in the message, because just the parameter name can be quite annoying if there are many methods that use the same parameter name.